platform/core/src/services/WorkflowStagesService/WorkflowStagesService.ts

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from 'react';
+import { CommandsManager } from '../../classes';
+import { ServicesManager, ToolbarService } from '../../services';
+import { PubSubService } from '../_shared/pubSubServiceInterface';
+
+export const EVENTS = {
+  ACTIVE_STAGE_CHANGED: 'event::workflowStagesService:activateStageChanged',
+  STAGES_CHANGED: 'event::workflowStagesService:stagesChanged',
+};
+
+export type WorkflowStage = {
```

Comment:
please add description on what is a workflowStage and what does it to and what is it capable of doing with examples here. We will have the workflowStage markdown documentation I guess later